README.md

```diff
@@ -34,36 +35,160 @@ npm install --save @webex/components @webex/sdk-component-adapter
 
 ## Usage
 
+### Setup
+
+When using the Webex SDK Adapter, it is expected to have a fully authenticated SDK instance passed to it:
+
+```js
+const webex = new Webex({
+  credentials: `<YOUR WEBEX ACCESS_TOKEN>`,
+});
+
+const adapter = new WebexSDKAdapter(webex);
+```
+
+#### Connect
+
+The Webex SDK requires different connections to be setup in order for it to be usable.
+Some examples of these connections include device registration and web socket connections.
+
+These connections are handled by the SDK Adapter, but require the `connect` function to be manually called.
+
+```js
+const webex = new Webex({
+  credentials: `<YOUR WEBEX ACCESS_TOKEN>`,
+});
+
+const adapter = new WebexSDKAdapter(webex);
+
+adapter.connect();
```

Comment:
I think we should let the user know that they need to wait for this promise to be resolved. by using `async/await`. Otherwise the devices will not be registered properly.